collect_app/src/main/java/org/odk/collect/android/provider/FormsProviderAPI.java

```diff
@@ -43,6 +43,7 @@ private FormsColumns() {
         public static final String CONTENT_ITEM_TYPE = "vnd.android.cursor.item/vnd.odk.form";
 
         // These are the only things needed for an insert
+        public static final String ID = "_id";
```

Comment:
Yes, that is true. Because, FormsColumns, which inherits BaseColumns does contain '_ID'.